app/javascript/components/upload/GeneListStep.jsx

```diff
@@ -39,7 +44,7 @@ function GeneListForm({
       <div className="col-md-12">
         <div className="form-terra">
           <p>
-            A list of genes and any computed expression values (mean, median, etc.) across any clusters.
+            A list of genes and any computed expression values (mean, median, etc.) across any clusters, for visualization as a heatmap
```

Comment:
Does this need a period?

Reply:
Hah, I could go either way on this, Eric recommended the period be removed